### PR TITLE
Add knowledge of streams to the quic-client-fuzzer

### PR DESCRIPTION
**Pull request by mattcaswell**
Allow the quic client fuzzer to create and accept streams, and swap between them.

### CODE REVIEW

File: fuzz/quic-client.c [hunk 1]
```diff
@@ -99,31 +111,112 @@ int FuzzerTestOneInput(const uint8_t *buf, size_t len)
...
+                case 0x00:
+                    if (state == READING)
+                        state = ACCEPTING_STREAM;
+                    break;
+                case 0x01:
+                    if (state == READING)
+                        state = CREATING_STREAM;
+                    break;
+                case 0x02:
+                    assert(0);
```
Comment (paulidale) [inline fuzz/quic-client.c:138]: Why is this here??

Comment (mattcaswell) [reply fuzz/quic-client.c]: Darn...debug leftover. Removed.

**Review by hlandau — APPROVED** [review]

**Review by t8m — APPROVED** [review]

**Comment by mattcaswell** [comment]
Pushed to master and 3.2.
